contact.js

```diff
@@ -0,0 +1,24 @@
+const form = document.getElementById('contact-form');
+const email= document.getElementById("gettouch");
+const id = document.getElementById("email");
+const EMAIL_CASE_CHECK = "Please Enter Email Address In Lower Case";
+const span = document.createElement("span");
+email.insertAdjacentElement('beforebegin', span);
+
+form.addEventListener("submit",function(event){
+    let email_add=(id.value);
+    var validRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+    if(email_add.match(validRegex) && (email_add == email_add.toLowerCase()))
+    {
+        alert("correct format and in lowercase");
+    }
+    else {
+       /*  alert("not in correct format"); */
+        event.preventDefault();
+        span.innerHTML = EMAIL_CASE_CHECK;
+        span.style.display = "block";
+        span.style.color="red";
+        /* form.appendChild(span); */
+        
+    }
+})
```

Comment:
- [ ] Good job so far, however, validation doesn't seem to work. As you can see below when I enter email in Uppercase then there is no error and the form submits. 
- [ ] Make sure If a user submits the contact form with the email field containing uppercase characters an error message should show up.
- [ ] If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.

https://user-images.githubusercontent.com/70269507/190104687-a0483629-d757-4cd8-a94b-7113495ea085.mov